### PR TITLE
[ADD] Add web_fetchmail_no_followers module

### PR DESCRIPTION
**Pull request by cmsalmeida**
This module don't add mail author on followers

### CODE REVIEW

File: web_mail_no_followers/README.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Description
```
Comment (hbrunn) [inline web_mail_no_followers/README.rst:1]: use OCA's template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by hbrunn** [comment]
please add tests

File: web_mail_no_followers/model/mail_thread.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    thread_id = message_post_super(self, cr, uid, thread_id, body=body,
+                                   subject=subject, type=type,
+                                   subtype=subtype, parent_id=parent_id,
+                                   attachments=attachments, context=nosub_ctx,
+                                   content_subtype=content_subtype, **kwargs)
+    return thread_id
+
+
+class MailNoFollowers(models.Model):
```
Comment (hbrunn) [inline web_mail_no_followers/model/mail_thread.py:54]: `mail.thread` is an abstract model, do the same

File: web_mail_no_followers/model/mail_thread.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+##############################################################################
+
+from openerp.addons.mail.mail_thread import mail_thread
+from openerp import models, api
+
+
+message_post_super = mail_thread.message_post
+
+
+@api.cr_uid_ids_context
```
Comment (hbrunn) [inline web_mail_no_followers/model/mail_thread.py:31]: why not v8 api?

**Comment by hbrunn** [comment]
I think this PR should be moved to crm or server-tools with a matching name change: `fetchmail_no_followers`

**Comment by pedrobaeza** [comment]
Or maybe social

**Pull request closed without merging**